tests/inputs/input.DSHAPE_desc

```diff
@@ -1,11 +1,12 @@
 # This DESC input file was auto generated from the VMEC input file
-# /DESC/tests/inputs/input.DSHAPE
+# .//tests//inputs//input.DSHAPE
+# on 10/27/2024 at 20:00:46.
```

Comment:
with this additional line, you may need to change the compared lines in `test_vmec_input`